homeassistant/components/lutron_caseta/cover.py

```diff
@@ -78,3 +68,79 @@ async def async_set_cover_position(self, **kwargs: Any) -> None:
         if ATTR_POSITION in kwargs:
```

Comment:
This guard is not needed since it can't be missing https://github.com/home-assistant/core/blob/4da701a8e9ee880fbbfbaec24c237e1e6146a509/homeassistant/components/cover/__init__.py#L181

This is an existing problem though

Reply:
no problem I can clean it while I'm here